src/Address.java

```diff
@@ -0,0 +1,57 @@
+public class Address {
+
+    // *********** Fields ************
```

Comment:
now it's completely fine to have these comments and sections, they'll help you remember order of things inside a class. However, just a general note for the future, that later we will avoid this, since it's not a good practice

Reply:
Acknowledged.